sdk/examples/intkey_java/IntegerKeyHandler.java

```diff
@@ -80,6 +82,60 @@ public String getVersion() {
     return namespaces;
   }
 
+  /**
+   * Helper function to decode the Payload of a transactions.
+   * Convert the co.nstant.in.cbor.model.Map to a HashMap.
+   */
+  public Map decodePayload(byte[] bytes) throws CborException {
+    ByteArrayInputStream bais = new ByteArrayInputStream(bytes);
+    co.nstant.in.cbor.model.Map data =
+        (co.nstant.in.cbor.model.Map) new CborDecoder(bais).decodeNext();
+    DataItem[] keys = data.getKeys().toArray(new DataItem[0]);
+    Map<String, String> result = new HashMap();
+    for (int i = 0; i < keys.length; i++) {
+      result.put(
+          keys[i].toString(),
+          data.get(keys[i]).toString());
+    }
+    return result;
+  }
+
+  /**
+   * Helper function to decode State retrieved from the address of the name.
+   * Convert the co.nstant.in.cbor.model.Map to a HashMap.
+   */
+  public  Map decodeState(byte[] bytes) throws CborException {
```

Comment:
Should return `Map<String, Integer>`

This would show that they are separate functions.